src/xcode/ENA/ENA/Source/Scenes/SelectValueViewController/Cell/SelectValueTableViewCell.swift

```diff
@@ -24,6 +24,13 @@ final class SelectValueTableViewCell: UITableViewCell {
 
 	func configure(_ cellViewModel: SelectValueCellViewModel) {
 		selectableValueLabel.text = cellViewModel.text
+		if let subtitle = cellViewModel.subtitle {
+			selectableSubtitleLabel.text = subtitle
+			selectableSubtitleLabel.isHidden = false
+			NSLayoutConstraint.activate([contentView.heightAnchor.constraint(greaterThanOrEqualToConstant: 62)])
```

Comment:
This might break. Making the subtile visible only by increasing height of the content view will not layout correct.